### PR TITLE
Update fees to current constants

### PR DESCRIPTION
**Pull request by djuanit0x**
Updated #1488

### CODE REVIEW

File: packages/api-derive/src/contracts/fees.ts [hunk 1]
```diff
@@ -47,12 +48,13 @@ function queryConstants (api: ApiInterfaceRx): Observable<ResultV2> {
 export function fees (api: ApiInterfaceRx): () => Observable<DeriveContractFees> {
   return memo((): Observable<DeriveContractFees> => {
     return queryConstants(api).pipe(
-      map(([creationFee, transferFee, callBaseFee, contractFee, rentByteFee, rentDepositOffset, tombstoneDeposit, transactionBaseFee, transactionByteFee]): DeriveContractFees => ({
+      map(([creationFee, callBaseFee, contractFee, rentByteFee, rentDepositOffset, surchargeReward, tombstoneDeposit, transactionBaseFee, transactionByteFee, transferFee]): DeriveContractFees => ({
```
Comment (jacogr) [inline packages/api-derive/src/contracts/fees.ts:51]:
```suggestion
      map(([callBaseFee, contractFee, creationFee,  transactionBaseFee, transactionByteFee, transferFee, rentByteFee, rentDepositOffset, surchargeReward, tombstoneDeposit]): DeriveContractFees => ({
```

Comment (djuanit0x) [reply packages/api-derive/src/contracts/fees.ts]: Reordered it. Thanks for the feedback.

File: packages/api-derive/src/contracts/fees.ts [hunk 2]
```diff
@@ -47,12 +48,13 @@ function queryConstants (api: ApiInterfaceRx): Observable<ResultV2> {
 export function fees (api: ApiInterfaceRx): () => Observable<DeriveContractFees> {
   return memo((): Observable<DeriveContractFees> => {
     return queryConstants(api).pipe(
-      map(([creationFee, transferFee, callBaseFee, contractFee, rentByteFee, rentDepositOffset, tombstoneDeposit, transactionBaseFee, transactionByteFee]): DeriveContractFees => ({
+      map(([callBaseFee, contractFee, creationFee, rentByteFee, rentDepositOffset, surchargeReward, tombstoneDeposit, transactionBaseFee, transactionByteFee, transferFee]): DeriveContractFees => ({
```
Comment (jacogr) [inline packages/api-derive/src/contracts/fees.ts:51]:
```suggestion
      map(([callBaseFee, contractFee, creationFee, transactionBaseFee, transactionByteFee, transferFee, rentByteFee, rentDepositOffset, surchargeReward, tombstoneDeposit]): DeriveContractFees => ({
```

This needs to match with the order above. the transaction* fees are out of order.

Comment (djuanit0x) [reply packages/api-derive/src/contracts/fees.ts]: My bad. I previously thought the args order need to be sorted alphabetically. Now, I surely rearrange the args in the right way :)

**Pull request merged**